seccomp.json

```diff
@@ -169,6 +171,7 @@
 				"ioprio_get",
 				"ioprio_set",
 				"ipc",
+				"keyctl",
```

Comment:
I am worried about this one. It might work on a modern system where the keyring is namespaced but the code will run on old systems as well where containers can now access the host keyring.

I think this must be a conditional for a longer period.

Reply:
To iterate on "a conditional": I think this should be added by the container engine if it's running a sufficiently recent kernel.

Reply:
Since this is upstream, we should allow it.  If a distribution is shipping an older kernel without support then the seccomp.json shipped by the distro should drop it.

We know we have other things in this seccomp.json file that break older versions of runc.

Reply:
https://lwn.net/Articles/779895/ indicates that this feature went into the upstream kernel 1.5 years ago.

Reply:
We would ship it into RHEL. Is the kernel supporting it?

Reply:
I don't know.  I will ask the author.  But we don't ship the seccomp.json file from here directly into RHEL, although we could accidently.